packages/docs/docs/auth/access-control.md

```diff
@@ -2,7 +2,7 @@
 sidebar_position: 4
 ---
 
-# Access Controls
+# Access Policies
```

Comment:
The original title "Access Controls" was intentional -- I think of "Access Controls" as a general concept, and "AccessPolicy" as a Medplum-specific concept.  They are different, and this page is about Access Controls more generally.

Sorry if that's too pedantic

I'd also prefer that we keep (1) titles, (2) filenames, and (3) URL slugs all consistent.

Reply:
Got it - yeah, let's use general concepts.  At a high level these should have a mental "How To" in the front, so I'm going to rename to "Enable Access Controls"